File: JANUARY/12-01-2026/1266.kt
class Solution {
    fun minTimeToVisitAllPoints(points: Array<IntArray>): Int {
        var ans = 0
        for (i in 1 until points.size) {
            val dx = kotlin.math.abs(points[i][0] - points[i - 1][0])
            val dy = kotlin.math.abs(points[i][1] - points[i - 1][1])
            ans += maxOf(dx, dy)
        }
        return ans
    }
}

File: JANUARY/12-01-2026/1266.rb
def min_time_to_visit_all_points(points)
  ans = 0
  (1...points.length).each do |i|
    dx = (points[i][0] - points[i - 1][0]).abs
    dy = (points[i][1] - points[i - 1][1]).abs
    ans += [dx, dy].max
  end
  ans
end